20_classes_lab/01.py

```diff
@@ -4,3 +4,24 @@
 Complete the class so correct result is printed
 """
 
+class Summer(object):
+    def __init__(self):
+        self._sum = 0
+    def add(self, *nums):
```

Comment:
Python's style guide suggests to leave a blank line just before method definitions (and that also looks better). Here's the relevant paragraph from it:
https://www.python.org/dev/peps/pep-0008/#blank-lines